helm/superset/values.yaml

```diff
@@ -227,6 +227,21 @@ ph-ee-engine:
     SPRING_PROFILES_ACTIVE: "bb"
     hostname: "mojaloop.sandbox.fynarfin.io"
     DFSPIDS: "gorilla,lion"
+    switch:
+      quotes:
+        host: "http://quoting-service.sandbox.fynarfin.io"
+        service: "quoting-service.sandbox.fynarfin.io"
```

Comment:
can these service use service names rather than ingress to communicate each other?

Reply:
not sure, need to try. Right now I am focusing on more important things. So I am aware of this but not paying much attention since its just a config.